ostelco-ios-client/ViewControllers/Login/LoginViewController.swift

```diff
@@ -53,28 +57,46 @@ class LoginViewController: UIViewController {
     private func configureButtonTitle(for index: Int) {
         if index == (OnboardingPage.allCases.count - 1) {
             // This is the last page.
-            primaryButton.setTitle(NSLocalizedString("Sign In", comment: "Title for sign in button."), for: .normal)
+            authorizationButton!.cornerRadius = primaryButton.intrinsicContentSize.height / 2
+            primaryButton.isHidden = true
+            buttonStackView.isHidden = false // Show "Sign In With Apple"
+            primaryButton.setTitle(NSLocalizedString("Sign In", comment: "Title for sign in button."), for: .normal) // Used for Analytics
         } else {
             primaryButton.setTitle(NSLocalizedString("Next", comment: "Action button in Carousel"), for: .normal)
+            primaryButton.isHidden = false
+            buttonStackView.isHidden = true
         }
     }
-    
+
+    func setupProviderLoginView() {
+        authorizationButton = ASAuthorizationAppleIDButton(type: .signIn, style: .whiteOutline)
```

Comment:
I got an compiler error here, not sure why, but locally the `signUp` type does not exist for me

<img width="259" alt="Screen Shot 2019-08-16 at 1 43 39 PM" src="https://user-images.githubusercontent.com/2212037/63165617-3dd56500-c02c-11e9-897b-f49e620a09fa.png">

Reply:
I see a different definition 
<img width="527" alt="Screenshot 2019-08-16 at 14 00 33" src="https://user-images.githubusercontent.com/370827/63166319-58a8d900-c02e-11e9-945d-f7c13289264c.png">

Reply:
No issues, I didn't have the latest xcode-beta